### PR TITLE
i18n: Add missing localization strings

### PR DESCRIPTION
**Pull request by 0neGal**
Due to the change in the formatting of localization files, or well, possible formatting, the old formatting is still allowed, but I digress, it'll be easier for everybody to simply make a comment here with the translated strings, and I'll paste them in not long after.

 - [x] German - @DxsSucuk
 - [x] French - @Alystrasz
 - [x] Spanish - @XNovaDelta

Strings to localize:
```json
"gui.settings.updatebuttons.title": "Repair actions",
"gui.settings.updatebuttons.desc": "If you're having problems with updates, some of these buttons may help in fixing them.",
"gui.settings.updatebuttons.buttons.reset_cached_api_requests": "Reset cached API requests",
"gui.settings.updatebuttons.buttons.force_northstar_reinstall": "Force re-install of Northstar",
"gui.settings.updatebuttons.buttons.force_delete_install_cache": "Force delete cached install files",
"gui.settings.miscbuttons.title": "Misc repair actions",
"gui.settings.miscbuttons.desc": "If you're having problems then some of these buttons may help fixing it",
"gui.settings.miscbuttons.buttons.open_gamepath": "Open gamepath",
"gui.settings.miscbuttons.buttons.reset_config": "Reset config file",
"gui.settings.miscbuttons.buttons.restart_viper": "Restart Viper",
"gui.settings.miscbuttons.buttons.change_gamepath": "Change gamepath",
"gui.settings.miscbuttons.buttons.force_quit_game": "Force quit Titanfall and Northstar",
"gui.settings.miscbuttons.buttons.force_quit_origin": "Force quit Origin and or EA Desktop",
"gui.settings.miscbuttons.open_gamepath_alert": "No valid gamepath is selected, so there's no gamepath to open, please select a valid gamepath first!",
"gui.settings.miscbuttons.reset_config_alert": "Please confirm that you want to reset the config file, after confirming Viper will delete the config file, and then restart.",
"gui.gamepath.lost_perms": "Your user seems to have lost permissions to create and or read files in the selected gamepath, please select a gamepath or gain access to read and write to the folder:\n\n",
"gui.gamepath.found_missing_perms": "Automatically found a valid gamepath, however your user doesn't have permissions to create and or read files in it, please manually select a different gamepath or gain access to read and write to the folder:\n\n",
"gui.gamepath.missing_perms": "Your user doesn't have permissions to create and or read files in the selected gamepath, please select a different gamepath or gain access to read and write to the folder:\n\n",
"ns.menu.force_quit": "Force quit game"
```

### CODE REVIEW

**Comment by DxsSucuk** [comment]
so post a comment with the localized strings in a codeblock instead of making a pr?

**Comment by 0neGal** [comment]
> so post a comment with the localized strings in a codeblock instead of making a pr?

Feel free to make a PR if you want, you'll just have to search through the localization file for the various strings, since the location now matters. Instead of having `object.object.string` we now have a proper JSON object, i.e `"object": {"object": {string: ""}}`, which on a larger basis makes the files far easier to navigate and more organized.

It does however have the side effect of requiring that `object.string1` and `object.string2` both be in the same object.

In the future I'll make a tiny script in `scripts/` to auto format/order the strings for the other files according to `en.json`

**TL;DR:** Either or, but I'd say a comment would be easier for you currently.

**Comment by XNovaDelta** [comment]
Here are the translated strings:
```
"gui.settings.updatebuttons.title": "Opciones de reparación",
"gui.settings.updatebuttons.desc": "Si estás teniendo problemas con las actualizaciones, algunos de estos botones pueden ayudar a solucionarlos.",
"gui.settings.updatebuttons.buttons.reset_cached_api_requests": "Restablecer las solicitudes de API en caché",
"gui.settings.updatebuttons.buttons.force_northstar_reinstall": "Forzar la reinstalación de Northstar",
"gui.settings.updatebuttons.buttons.force_delete_install_cache": "Forzar la eliminación de archivos de instalación en caché",
"gui.settings.miscbuttons.title": "Acciones de reparación diversas",
"gui.settings.miscbuttons.desc": "Si estás teniendo problemas, entonces algunos de estos botones pueden ayudar a solucionarlo",
"gui.settings.miscbuttons.buttons.open_gamepath": "Abrir ruta del juego",
"gui.settings.miscbuttons.buttons.reset_config": "Restablecer archivo de configuración",
"gui.settings.miscbuttons.buttons.restart_viper": "Reiniciar Viper",
"gui.settings.miscbuttons.buttons.change_gamepath": "Cambiar ruta del juego",
"gui.settings.miscbuttons.buttons.force_quit_game": "Cerrar forzosamente Titanfall y Northstar",
"gui.settings.miscbuttons.buttons.force_quit_origin": "Cerrar forzosamente Origin y/o EA Desktop",
"gui.settings.miscbuttons.open_gamepath_alert": "No se ha seleccionado una ruta de juego válida, por lo que no hay una ruta de juego para abrir. ¡Por favor, selecciona primero una ruta de juego válida!",
"gui.settings.miscbuttons.reset_config_alert": "Por favor, confirma que deseas restablecer el archivo de configuración. Después de confirmar, Viper eliminará el archivo de configuración y luego reiniciará.",
"gui.gamepath.lost_perms": "Tu usuario parece haber perdido permisos para crear o leer archivos en la ruta de juego seleccionada. Por favor, selecciona una ruta de juego válida o recupera el acceso para leer y escribir en la carpeta:\n\n",
"gui.gamepath.found_missing_perms": "Se encontró automáticamente una ruta de juego válida, pero tu usuario no tiene permisos para crear o leer archivos en ella. Por favor, selecciona manualmente una ruta de juego diferente o obtén acceso para leer y escribir en la carpeta:\n\n",
"gui.gamepath.missing_perms": "Tu usuario no tiene permisos para crear o leer archivos en la ruta de juego seleccionada. Por favor, selecciona una ruta de juego diferente o obtén acceso para leer y escribir en la carpeta:\n\n",
"ns.menu.force_quit": "Cerrar forzosamente el juego"
```

**Comment by DxsSucuk** [comment]
```"gui.settings.updatebuttons.title": "Reparations Aktionen",
"gui.settings.updatebuttons.desc": "Bei Problemen mit updates könnte diese Aktionen helfen.",
"gui.settings.updatebuttons.buttons.reset_cached_api_requests": "API zwischenspeicher leerung erzwingen",
"gui.settings.updatebuttons.buttons.force_northstar_reinstall": "Northstar neuinstallation erzwingen",
"gui.settings.updatebuttons.buttons.force_delete_install_cache": "Zwischengespeicherte Installationsdateien löschung erzwingen",
"gui.settings.miscbuttons.title": "Zusatz Aktionen",
"gui.settings.miscbuttons.desc": "Beim weiteren Problemen könnten diesen Aktion helfen.",
"gui.settings.miscbuttons.buttons.open_gamepath": "Spielepfad öffnen",
"gui.settings.miscbuttons.buttons.reset_config": "Konfiguration zurücksetzung erzwingen",
"gui.settings.miscbuttons.buttons.restart_viper": "Viper neustarten",
"gui.settings.miscbuttons.buttons.change_gamepath": "Spielepfad ändern",
"gui.settings.miscbuttons.buttons.force_quit_game": "Titanfall/Northstar schließung erzwingen",
"gui.settings.miscbuttons.buttons.force_quit_origin": "Origin/EA Desktop schließung erzwingen",
"gui.settings.miscbuttons.open_gamepath_alert": "Fehlerhafter Spielepfad, dadurch kann keiner geöffnet werden. Bitte wähle den richtigen Pfad zuerst!",
"gui.settings.miscbuttons.reset_config_alert": "Bestätige die Zurücksetzung der Konfiguration, Viper wird nach dem bestätigen die Konfiguration Datein löschen und neustarten.",
"gui.gamepath.lost_perms": "Wie es aussieht hat der jetzige Nutzer die Rechte Datein zulesen oder zuschreiben im Spielepfad verloren, bitte wähle einen richtigen Pfad aus oder erhalte die benötigten Berechtigungen für den jetzigen:\n\n",
"gui.gamepath.found_missing_perms": "Es wurde ein Spielepfad automatisch gefunden, jedoch fehlen dir die Rechte Datein zulesen oder zuschreiben in diesem Pfad. Bitte wähle einen anderen Pfad aus oder erhalte die benötigten Berechtigungen für den gewählten Pfad:\n\n",
"gui.gamepath.missing_perms": "Wie es aussieht hat der jetzige Nutzer die Rechte Datein zulesen oder zuschreiben im Spielepfad nicht, bitte wähle einen richtigen Pfad aus oder erhalte die benötigten Berechtigungen für den gewählten Pfad:\n\n",
"ns.menu.force_quit": "Schließen erzwingen"```

sorry for the late send it!
